build/build.cake

```diff
@@ -1,177 +0,0 @@
-#module nuget:?package=Cake.LongPath.Module&version=1.0.1
-
-#addin nuget:?package=Cake.FileHelpers&version=4.0.1
-#addin nuget:?package=Cake.Powershell&version=1.0.1
-#addin nuget:?package=Cake.GitVersioning&version=3.4.244
-
-#tool nuget:?package=vswhere&version=2.8.4
-
-using System;
-using System.Linq;
-using System.Text.RegularExpressions;
-
-//////////////////////////////////////////////////////////////////////
-// ARGUMENTS
-//////////////////////////////////////////////////////////////////////
-
-var target = Argument("target", "Default");
-
-//////////////////////////////////////////////////////////////////////
-// PREPARATION
-//////////////////////////////////////////////////////////////////////
-
-var baseDir = MakeAbsolute(Directory("../")).ToString();
-var buildDir = baseDir + "/build";
-
-var Solution = baseDir + "/ColorCode.sln";
-var nupkgDir = buildDir + "/nupkg";
-
-string Version = null;
-
-//////////////////////////////////////////////////////////////////////
-// METHODS
-//////////////////////////////////////////////////////////////////////
-
-void VerifyHeaders(bool Replace)
```

Comment:
Since we're removing this step, we'll want to add/update the editor config to have the file header stuff like we did for Labs.